Графика для всех страниц

username1298 opened the pull request:
1. Экспорт изображений в подходящих форматах
2. Создание папки images и размещение изображений в ней (для удобства были созданы подпапки по типам изображений, по рекомендации из видео "3.6. Экспортируем графику")
3. Оптимизация изображений через сервисы Squoosh, SVGOMG
4. Подключение контентных изображений с использованием относительных адресов
5. Заполнение атрибутов alt у изображений карточек товаров
6. Сопоставление критериям задания

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2300403/tasks/7)

Commit:
Выполняет домашнее задание 3.16

1. Экспорт изображений в подходящих форматах
2. Создание папки images и размещение изображений в ней (для удобства были созданы подпапки по типам изображений, по рекомендации из видео "3.6. Экспортируем графику")
3. Оптимизация изображений через сервисы Squoosh, SVGOMG
4. Подключение контентных изображений с использованием относительных адресов
5. Заполнение атрибутов alt у изображений карточек товаров
6. Сопоставление критериям задания

AlexandrRumiantsev commented:
1) Некоторые изображения не указаны, по заданию нужно подключить все (https://dropmefiles.com/PG3Z8)
2) Дополнительно можно подключить фавиконку по критерию (https://up.htmlacademy.ru/htmlcss/36/criteries#HTML-08)

Commit:
Добавляет favicon

1. Экспортировал из макета favicon в PNG, оптимизировал через squoosh, сохранил в в папке images
2. 1. Экспортировал из макета favicon в SVG, сконвертировал в ICO, сохранил в корневом каталоге
3. Сделал разметку для favicon на всех страницах согласно критерию (https://up.htmlacademy.ru/profession/frontender/16/htmlcss/36/criteries#HTML-08)
